Troupon.Checkout.Core.Application/DTOs/OrderPlacedDto.cs

```diff
@@ -0,0 +1,9 @@
+using System;
+
+namespace Troupon.Checkout.Core.Application.DTOs
```

Comment:
I'm not sure, but I think we started to put DTO in 2 places. I saw them in 
- Core.Applicaiton.DTO
- Core.Domain.DTO

It was already like that when I checkout the project at the begining. But I'm used to have DTO dirrectly in the API layer.

Maybe we should talk about this in team and keep a convension.

So I don't ask you to move them, because I'm not too sure about this. But we should probably have a standard for it.

Reply:
Totally agree! I will try to add this as a topic of discussion for our next discussion meeting together